llvm/lib/Transforms/Utils/CodeLayout.cpp

```diff
@@ -62,6 +62,12 @@ cl::opt<bool> ApplyExtTspWithoutProfile(
     "ext-tsp-apply-without-profile",
     cl::desc("Whether to apply ext-tsp placement for instances w/o profile"),
     cl::init(true), cl::Hidden);
+
+namespace codelayout {
+cl::opt<unsigned>
+    CDMaxChainSize("cds-max-chain-size", cl::Hidden, cl::init(128),
```

Comment:
`cdsort-max-chain-size` maybe?

Reply:
Renamed to `cdsort-max-chain-size` in 9403e71c4270bf830384b43b5c8deff13e1bf3d3